.circleci/config.yml

```diff
@@ -80,18 +80,31 @@ commands:
         type: string
     steps:
       - run: mkdir test-cases/
-      - run: cd test-cases && python3 ../scripts/isolate_tests.py ../test/
-      - run: cd test-cases && python3 ../scripts/bytecodecompare/prepare_report.py << parameters.binary_path >> --interface standard-json --report-file "../bytecode-report-<< parameters.label >>-json.txt"
-      - run: cd test-cases && python3 ../scripts/bytecodecompare/prepare_report.py << parameters.binary_path >> --interface cli --report-file "../bytecode-report-<< parameters.label >>-cli.txt"
+      - run:
+          name: Prepare input files
+          command: |
+            cd test-cases/
+            python3 ../scripts/isolate_tests.py ../test/
+      - run:
+          name: Generate bytecode report
+          command: |
+            cd test-cases/
+            interface=$(echo -e "standard-json\ncli" | circleci tests split)
+            echo "Selected interface: ${interface}"
+
+            python3 ../scripts/bytecodecompare/prepare_report.py \
+              << parameters.binary_path >> \
+              --interface "$interface" \
+              --report-file "../bytecode-report-<< parameters.label >>-${interface}.txt"
       - store_artifacts:
-          path: bytecode-report-<< parameters.label >>-json.txt
+          path: bytecode-report-<< parameters.label >>-standard-json.txt
       - store_artifacts:
           path: bytecode-report-<< parameters.label >>-cli.txt
       - persist_to_workspace:
           root: .
           paths:
-            - bytecode-report-<< parameters.label >>-json.txt
-            - bytecode-report-<< parameters.label >>-cli.txt
+            - bytecode-report-<< parameters.label >>-*.txt
```

Comment:
Not sure how this works - can you explain?  I.e., now that we're not using explicit names, what does `*` do?

Reply:
It's just a glob like in the shell. It will pick up any files that match the pattern. I'm changing it because now each run of this command will only produce one of the artifacts.

IIRC `store_artifacts` will just ignore files that do not exist, while `persists_to_workspace` won't, but you can use glob patterns to work around that which I'm doing here.

Reply:
I just noticed that I converted both lines to globs instead of removing one. This will still work but is unnecessary. I'll push a correction in a moment.